cfgov/v1/__init__.py

```diff
@@ -35,12 +39,30 @@ def environment(**options):
         'choices_for_page_type': ref.choices_for_page_type,
         'is_blog': ref.is_blog,
         'get_page_state_url': share.get_page_state_url,
+        'external_links': external_links,
     })
     env.filters.update({
         'slugify': slugify,
     })
     return env
 
+EXTERNAL_LINK_PATTERN = '(https?:\/\/(?:www\.)?(?![^\?]*(cfpb|consumerfinance).gov)(?!(content\.)?localhost).*)'
+
+
+def external_links(value):
+    soup = BeautifulSoup(expand_db_html(value.source))
+
+    try:
+        for a in soup('a'):
+            p = re.compile(EXTERNAL_LINK_PATTERN)
```

Comment:
whoops yea i'll move that out of the loop. I thought a regex was the simplest way to handle anything outside of `cfpb.gov`, `consumerfinance.gov`, `localhost` and `content`.

i'm open to suggestions though?